DEVELOPING.md

```diff
@@ -0,0 +1,23 @@
+# Project Structure
+
+Here is a general overview of some of the more important directories in the project.
+
+```
+.
+├── data
```

Comment:
We can leave this in the Developing.md
I need to add this and put workspace.json into it. I'll do that next patch.